proto/Cargo.toml

```diff
@@ -12,14 +12,14 @@ rust-version = "1.67"
 [dependencies]
 hex = { version = "0.4" }
 miden-crypto = { git = "https://github.com/0xPolygonMiden/crypto", branch = "next" }
```

Comment:
May make sense to add `default-features = false` here.

Reply:
done